include/RAJA/Layout.hxx

```diff
@@ -55,9 +55,9 @@
 
 #include <iostream>
 #include <limits>
-#include "RAJA/IndexValue.hxx"
-#include "RAJA/LegacyCompatibility.hxx"
-#include "RAJA/foralln/Permutations.hxx"
+#include "RAJA/internal/IndexValue.hxx"
+#include "RAJA/internal/LegacyCompatibility.hxx"
+#include "RAJA/internal/foralln/Permutations.hxx"
```

Comment:
What is the policy you are applying to determine what goes into the 'RAJA/internal' include directory and what goes in the 'RAJA' include directory?